src/program.cpp

```diff
@@ -145,6 +145,31 @@ String RemoveCommas(String line)
     return returnString;
 }
 
+TokenList RemoveComment(TokenList tokens)
+{
+    TokenList tokensNoComments; 
+    for (size_t i = 0; i < tokens.size(); i++) {
```

Comment:
nit: For consistency let's keep the curly braces on their own lines

Reply:
fixed